### DISCUSSION TITLE
微信开发者工具无法预览

### DISCUSSION THREAD

**Discussion by ziyouchutuwenwu** [NervJS/taro#11338] [Q & A]
步骤如下，taro版本3.4.2, 系统linux, win下测试也是一样的，始终不行，但是npm实在是太慢了，所以用linux

```sh
taro init demo
npm run dev:weapp
```

微信开发者工具，导入项目根目录，设置详情里面，关闭下面这些

```sh
js编译成es5
上传代码样式自动补全
上传代码自动压缩样式
上传代码自动压缩脚本
上传代码自动压缩wxml
```

点击预览，提示

```sh
以下文件没有上传
taro.js.map
vendors.js.map
```

另外，预览是空白

菜单，微信开发者工具， 调试，调试微信开发者工具，截图如下
![1](https://user-images.githubusercontent.com/7458713/155246588-4e615890-3f77-4766-9de9-633982bd5f9b.png)
![2](https://user-images.githubusercontent.com/7458713/155246695-cab34d27-a07d-420f-aa36-ed3d1b40cd07.png)

**Comment by TheKonka** [comment]
方便的 话 可以给个repo

**Reply by Chen-jj** [reply]
+1
